src/layup_cmdline/orbitfit.py

```diff
@@ -98,6 +98,14 @@ def main():
         default="csv",
         required=False,
     )
+    optional.add_argument(
+        "--output_orbit",
+        help="the orbit format for the output file. [KEP, CART, COM, BKEP, BCART, BCART_EQ, BCOM]",
+        dest="ouptu_orbit",
```

Comment:
It looks like `output_orbit_format` is what is expected in orbitfit.py.
```suggestion
        dest="output_orbit_format",
```

Reply:
As discussed elsewhere, the actual command line argument being used is here 

https://github.com/Smithsonian/layup/blob/c8791d219269c7a19714fd7e50e636de11d86e9b/src/layup_cmdline/orbitfit.py#L131

Don't know how this made its way onto github since this change wasn't on my local branch. But successfully finagled this back out or the pr.